lib/tasks/dev.rake

```diff
@@ -1,13 +1,31 @@
+require 'csv'
+
 if Rails.env.development? || Rails.env.staging?
   namespace :dev do
-    desc "Seed data for development environment"
-    task prime: "db:setup" do
-      User.create(
-        first_name: "John",
-        last_name: "Wayne",
-        email: "john@example.org",
-        password: "stagecoach"
-      )
+    desc 'Seed data for development environment'
+    task prime: 'db:setup' do
+      if Rails.env.development?
+        User.create(
+          first_name: 'John',
+          last_name: 'Wayne',
+          email: 'john@example.org',
+          password: 'stagecoach'
+        )
+      end
+
+      CSV.foreach('lib/data/prime/directors.csv', headers: true) do |director|
+        Director.find_or_create_by!(
+          name: director['name'],
+          country: Country.find_by(code: director['country_code'])
+        )
+      end
+
+      CSV.foreach('lib/data/prime/movies.csv', headers: true) do |movie|
+        Movie.find_or_create_by!(
+          original_title: movie['original_title'],
+          directors: [Director.find_by(name: movie['director_name'])]
```

Comment:
Should we use `find_by!` here (and on line 19) so that, if there's a failure loading, we know exactly where it failed?

Line of thought: we should not try to find the movie if finding the director fails before.

Reply:
About your last remark: we're not really _finding_, we're "upserting".

But yeah, let's make it blow up if the """foreign key""" with the directors' names isn't matched (which, in this case, would mean that we have a mistake in our CSVs).

Reply:
Check again, please? @gdiasdasilva